### DISCUSSION TITLE
TFT35 E3 V3.0 PID calibration

### DISCUSSION THREAD

**Discussion by Grzesinski** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2335] [Q&A]
Has anyone used PID calibration in the TFT35 E3 V3.0 display (with SKR MINI E3 V2.0) ?
From the menu, you can choose what the calibration should concern (hot-end, bed) and the target temperature. Looks good.
But regardless of the choice, the head is always heated and the process always ends with an error.
Can someone confirm that this function is working properly?
Now I am calibrating directly from the TFT terminal using M commands and it is working properly.

**Comment by Infactionfreddy** [comment]
Yes i have the same problem on the TFT70 V3.0

**Reply by Grzesinski** [reply]
> Yes i have the same problem on the TFT70 V3.0
Resolved
PID wizzard in TFT screen need 
#define PIDTEMPBED in Configuration.h in current Marlin

Now all works properly.
